src/main/webapp/login.html

```diff
@@ -44,7 +44,7 @@
                 <div class="card card-signin my-5">
                     <div class="card-body">
                         <h5 class="card-title text-center">Sign In</h5>
-                        <form class="form-signin", method = post, action="LoginServlet">
+                        <form class="form-signin" action="login" method=post>
```

Comment:
I don't really understand those changes and why they are working. I guess we don't have time to go over? But I feel like I should not be the one approving if I don't fully understand